packages/atlas/src/components/ListItem/ListItem.styles.ts

```diff
@@ -51,7 +57,8 @@ export const Container = styled('div', { shouldForwardProp: isPropValid })<Conta
   align-items: center;
   user-select: none;
   cursor: pointer;
-  background-color: ${({ highlight }) => (highlight ? cVar('colorBackgroundMutedAlpha') : 'unset')};
+  background-color: ${({ highlight, highlightWhenActive }) =>
```

Comment:
Can we have only `highlight` here? Why do we need another prop? We could pass on the component level just `highlight || highlightWhenActive`? Am I right?

Reply:
I agree 👍